packages/graphql-server-hapi/src/hapiApollo.test.ts

```diff
@@ -1,10 +1,10 @@
 import * as hapi from 'hapi';
-import { graphqlHapi, graphiqlHapi, HapiPluginOptions } from './hapiApollo';
+import { graphqlHapi, graphiqlHapi } from './hapiApollo';
 import 'mocha';
 
 import testSuite, { Schema } from 'graphql-server-integration-testsuite';
 
-function createApp(createOptions: HapiPluginOptions) {
+function createApp(createOptions) {
```

Comment:
Aha, that worked. I had to cast and check for the presence of .graphqlOptions and .graphiqlOptions further below.